sunpy/map/mapsequence.py

```diff
@@ -207,8 +207,14 @@ def updatefig(i, im, annotate, ani_data, removes):
                 im.axes.reset_wcs(ani_data[i].wcs)
                 wcsaxes_compat.default_wcs_grid(axes)
             else:
-                im.set_extent(np.concatenate((ani_data[i].xrange.value,
-                                              ani_data[i].yrange.value)))
+                xmin = ani_data[i].center.data.lon - ani_data[i].dimensions[0] / 2. * ani_data[i].scale[0]
+                xmax = ani_data[i].center.data.lon + ani_data[i].dimensions[0] / 2. * ani_data[i].scale[0]
+                anidata_xrange = u.Quantity([xmin, xmax])
+                ymin = ani_data[i].center.data.lat - ani_data[i].dimensions[1] / 2. * ani_data[i].scale[1]
+                ymax = ani_data[i].center.data.lat + ani_data[i].dimensions[1] / 2. * ani_data[i].scale[1]
+                anidata_yrange = u.Quantity([ymin, ymax])
+                im.set_extent(np.concatenate((anidata_xrange.value,
+                                              anidata_yrange.value)))
```

Comment:
Since `xrange` and `yrange` were deleted, I just copied their code into here since it was the only place it was used.